applications/nrf5340_audio/src/bluetooth/bt_management/scanning/bt_mgmt_scan_for_broadcast.c

```diff
@@ -42,6 +42,33 @@ struct broadcast_source {
 	uint32_t broadcast_id;
 };
 
+static void scan_restart_worker(struct k_work *work)
+{
+	int ret;
+
+	/* Delete pending PA sync before restarting scan */
+	ret = bt_mgmt_pa_sync_delete(pa_sync);
+	if (ret) {
+		LOG_WRN("Failed to delete pending PA sync: %d", ret);
```

Comment:
Should do LOG_ERR here and return?

Reply:
I'd rather end up in a scanning state than in between